webapp/client/src/components/FormFieldDropDown.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import FieldLabel from "./FieldLabel";
+import selectIcon from "../assets/icons/select_icon.svg";
+
+const DropDown = styled.div`
+  & .steuerlotse-select {
+    border: 2px solid var(--border-color);
+    border-radius: 0;
+    background-image: url(${selectIcon});
+    background-repeat: no-repeat;
+    background-size: 0.75rem;
+    min-height: 55px;
+  }
+
+  & .steuerlotse-select:focus {
+    border: 2px solid var(--focus-border-color);
+    box-shadow: 0 0 0 2px var(--focus-color);
+  }
+`;
+
+function FormFieldDropDown({
+  fieldName,
+  fieldId,
+  options,
+  defaultOption,
+  selectedValue,
+  required,
+  label,
+  details,
+  errors,
+  onChangeHandler,
+}) {
```

Comment:
Why did you decide to pass the `onChangeHandler` here? Do you expect it to be different for different drop downs? Otherwise we could just declare it for this component, riht?

Reply:
The onChangeHandler is needed to change the state in the parent component. Therefore, it should be adaptable and we are not able to set it in this component because we need it to change the state in the parent component.
That is why we have to pass it

Reply:
Ah..right, I see, thanks! Do we need it onBlur and onChange though? Wouldn't onChange be enough? Probably just asking dumb questions here. :D

Reply:
Not at all a dumb question. When I was developing this feature we had an older version of `eslint-plugin-jsx-a11y` in use which enforced the following rule: [jsx-a11y/no-onchange](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md) - use onBlur on select events because some browser do not support onChange correctly for selects

That rule has been deprecated and we now use a newer version of the plugin that does not enforce the rule. Therefore, we could get rid of onBlur, now.